Bowling/NUnitTestBowling/BowlingTests.cs

```diff
@@ -0,0 +1,70 @@
+using NUnit.Framework;
+using System.Runtime.InteropServices;
+using BowlingApp;
+using System.IO;
+using System;
+using System.Collections.Generic;
+
+namespace NUnitTestBowling
+{
+    public class Tests
+    {
+
+        [Test]
+        [TestCase("XXXXXXXXXXXX", "300", Description = "Perfect Game")]
+        [TestCase("9-9-9-9-9-9-9-9-9-9-", "90", Description = "Open Frames with -")]
+        [TestCase("5/5/5/5/5/5/5/5/5/5/5", "150", Description = "All Spares")]
+        [TestCase("548/XX725/7/52X6/5", "158", Description = "Random Open Spares Strikes")]
+        [TestCase("54637281233445625381", "82", Description = "All Open")]
+        public void TestCompleteScoringOfInputString(string marks, string expectedScore)
+        {
+            using (var sw = new StringWriter())
+            {
+                // user string for bowling record
+                using (var sr = new StringReader(marks))
+                {
+                    Console.SetOut(sw);
+                    Console.SetIn(sr);
+
+                    Bowling bowling = new Bowling();
```

Comment:
Since this is used in every test in this file, it looks like a good candidate to be an instance variable of the test class; checkout the [SetUpAttribute](https://docs.nunit.org/articles/nunit/writing-tests/attributes/setup.html) for information on how to initialize it before each test.